libraries/AC_Planck/AC_Planck.cpp

```diff
@@ -286,3 +333,76 @@ bool AC_Planck::get_posvel_cmd(Location &loc, Vector3f &vel_cms, float &yaw_cd,
   _cmd.is_new = false;
   return true;
 }
+
+bool AC_Planck::check_if_high_tension_failed(const int32_t alt_cm) {
+  //No comms from the tether
+  bool tether_comms_failed = is_tether_timed_out();
+
+  //Determine if high tension has been or should have been triggered
+  bool high_tension_triggered = _tether_status.high_tension || tether_comms_failed;
```

Comment:
Technically, technically, high tension is triggered 5 seconds after a time out, so we should probably include that here

Reply:
Thats what `tether_comms_failed` covers.

Reply:
Actually, wait never mind, it goes into high tension 5 seconds after loosing comms, not 5 seconds after the failsafe is triggered